src/main/java/com/okta/sdk/clients/SessionApiClient.java

```diff
@@ -68,14 +64,22 @@ public Session createSessionWithSessionTokenAndAdditionalFields(String sessionTo
     }
 
     public Session validateSession(String sessionId) throws IOException {
-        return get(getEncodedPath("/%s", sessionId), new TypeReference<Session>() { });
+        return get(getEncodedPath("/%s", sessionId), new TypeReference<Session>() {
+        });
     }
 
     public Session extendSession(String sessionId) throws IOException  {
-        return put(getEncodedPath("/%s", sessionId), new TypeReference<Session>() { });
+        return put(getEncodedPath("/%s", sessionId), new TypeReference<Session>() {
+        });
     }
 
     public void clearSession(String sessionId) throws IOException {
         delete(getEncodedPath("/%s", sessionId));
     }
+
+    @Override
+    protected String getFullPath(String relativePath) {
+        return String.format("/api/v%d/sessions%s", this.apiVersion, relativePath);
+    }
+
 }
```

Comment:
nit: nit: That's actually intentional, see for example the implementation of the String class.